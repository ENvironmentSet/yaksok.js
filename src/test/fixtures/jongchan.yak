아이: 10
어른: 20
종찬: 22

약속 (입력)이/가/이가 어린가
    결과: 입력 < 15

(아이 가 어린가) 보여주기 # '아이'와 '가'를 붙이면 '아'라는 변수를 찾게되므로 에러.
(어른이 어린가) 보여주기
(종찬이가 어린가) 보여주기

약속 (입력1)이/가/이가 (입력2)보다/이보다 어린가/어린지
    결과: 입력1 < 입력2

(아이 가 종찬이보다 어린지) 보여주기