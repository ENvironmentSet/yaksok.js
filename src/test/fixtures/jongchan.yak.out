참
거짓
거짓
참
